components/logins/Cargo.toml

```diff
@@ -13,7 +13,7 @@ sync15 = { path = "../sync15" }
 serde = "1.0.79"
 serde_derive = "1.0.79"
 serde_json = "1.0.28"
-log = "0.4"
+log = "0.4.6"
```

Comment:
Is this needed, or just good form?  Can we pin a precise version in all of our `Cargo.toml` files while we're here?  See also #529, which feels similar.

Reply:
I'm not sure this was intentional, actually.